README.md

```diff
@@ -1,6 +1,28 @@
+# Pursuit-Core-Web-React-Lifecycle-Methods-Lab
+
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+## Tasks
+
+Build an app that has the following functionality:
+
+- Users can create a new Todo by typing text into an `input` inside a `form` and pressing enter
+- Shows a list of todos with id `#todos` underneath that input using instances of a `Todo` component
+- Users can delete an existing Todo by pressing an 'x' `button` next to it
+
+Implement lifecycle methods to fire the created/deleted notifications:
+
+- Use `componentDidMount` in `Todo` to show a `"success"` toast with the text `New todo added: {todo}`
+- Use `componentWillUnmount` in `Todo` to show an `"error"` toast with the text `Todo deleted: {todo}`
```

Comment:
👍  Thank you for giving the specific guidance here on which lifecycle methods to use!